tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/IisCheckTests.cs

```diff
@@ -71,20 +69,58 @@ public async Task WorkingApp(bool mixedRuntimes)
             }
         }
 
+        [SkippableFact]
+        public async Task OutOfProcess()
+        {
+            EnsureWindowsAndX64();
+
+            var buildPs1 = Path.Combine(EnvironmentTools.GetSolutionDirectory(), "tracer", "build.ps1");
+
+            try
+            {
+                // GacFixture is not compatible with .NET Core, use the Nuke target instead
+                Process.Start("powershell", $"{buildPs1} GacAdd --framework net461").WaitForExit();
+
+                using var iisFixture = CreateIisFixture();
+
+                iisFixture.TryStartIis(this, IisAppType.AspNetCoreOutOfProcess);
+
```

Comment:
nit - You could move that in the CreateIisFixture as you do it each time.

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2450/commits/af79d8e0c7ade08264b8af4fd136b2a41aa9639d

Reply:
Sorry I still have a hard time doing a multi line review it seems... 
I meant to set this comment on this line and the few above where you warmup the app
```
                // Send a request to initialize the app
                using var httpClient = new HttpClient();
                await httpClient.GetAsync($"http://localhost:{iisFixture.HttpPort}/");
```

Still a nit though